File: popsicles/Popsicles.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int siblings, popsicles;
            siblings = Convert.ToInt32(Console.ReadLine());
            popsicles = Convert.ToInt32(Console.ReadLine());
            
            //your code goes here
            
            bool isEven = popsicles % siblings == 0;
            
            if (isEven) {
            	Console.WriteLine("give away");
            }
            else {
            	Console.WriteLine("eat them yourself");
            }
        }
    }
}

File: popsicles/Popsicles.cpp
#include <iostream>
using namespace std;

int main() {
    int siblings, popsicles;
    //take input
    cin>>siblings>>popsicles;

    //your code goes here
    
    cout << (popsicles % siblings ? "eat them yourself" : "give away");
        
    return 0;
}
